BoxContentSDKSampleApp/Cartfile

```diff
@@ -0,0 +1,2 @@
+# Box SDKs
+git "git@github.com:box/box-ios-sdk.git" ~> 1.0
```

Comment:
We could also use `github "box/box-ios-sdk" ~> 1.0`. I'm not sure if that affects whether Carthage tries to use binaries from Github releases, but maybe. Carthage would then use https by default, but I think that's fine for public repos.

Reply:
done